OutOfSchool/OutOfSchool.WebApi/Hubs/LocalDistributedHubLifetimeManager.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace OutOfSchool.WebApi.Hubs;
 
-public class LocalDistributedHubLifetimeManager<THub> : HubLifetimeManager<THub>, IDisposable
+public sealed class LocalDistributedHubLifetimeManager<THub> : HubLifetimeManager<THub>, IDisposable
```

Comment:
(Curiosity) I wonder why precisely this class is made `sealed`? No objections, just asking

Reply:
@VadymLevkovskyi  for the previous class (`CacheService`) we may have an inheriting class sometime, so i followed a recommended approach for a Disposable pattern there. For this class there will be no inheritance, so adding `sealed` makes it compliant to "implement Disposable pattern correctly i case someone will make a child class" warning in Sonar.

Reply:
Ah, `Disposable` pattern. Thanks!